cpp/src/parquet/column_reader_test.cc

```diff
@@ -356,6 +356,34 @@ TEST_F(TestPrimitiveReader, TestReadValuesMissing) {
                ParquetException);
 }
 
+// Repetition level byte length reported in Page but Max Repetition level
+// is zero for the column.
+TEST_F(TestPrimitiveReader, TestRepetitionLvlBytesWithMaxRepetitionZero) {
+  constexpr int batch_size = 4;
+  max_def_level_ = 1;
+  max_rep_level_ = 0;
+  NodePtr type = schema::Int32("a", Repetition::OPTIONAL);
+  const ColumnDescriptor descr(type, max_def_level_, max_rep_level_);
+  const std::array<uint8_t, 21> page_data{0x3,  0x3, 0x7, 0x80, 0x1, 0x4, 0x3,
```

Comment:
I don't think there is a strong reason to use array here, as it requires setting the actual number of bytes, you could just use a vector.